### ISSUE TITLE
Update Compliance was retired on March 31, 2023 - Whis workbook no longer work

### ISSUE THREAD

**Issue by dacool** [MSEndpointMgr/Reporting#17]
Update Compliance was retired on March 31, 2023 and the service has been removed. 
https://learn.microsoft.com/en-us/windows/deployment/update/update-compliance-monitor

Update Compliance has been replaced by Windows Update for Business reports. 
https://learn.microsoft.com/en-us/windows/deployment/update/wufb-reports-overview

LogAnalytics tables and schema has changed, and this workbook no longer work.

Any plans to make it support Windows Update for Business coming?

**Comment by Marshyp** [comment]
Agreed, it'd be amazing to see this moved to Windows Update for Business.
For what it's worth, some of the data seems to work but we're seeing anything referencing **WaaSDeploymentStatus** is broken. 

I tried to do a quick search to see what replaced this but don't seem to be getting very far. Will keep digging.

**Comment by jjhfarmer** [comment]
I'd also be interested to know whether we can replace `WaaSDeploymentStatus` with another expression. Aside from that and `WUDOStatus` and `WUDOAggregatedStatus`, the Workbook still seems to work fine. 

This is the new schema table but it seems like it doesn't have anything directly related to the deprecated expressions: https://learn.microsoft.com/en-us/windows/deployment/update/wufb-reports-schema

**Comment by Marshyp** [comment]
Agreed @jjhfarmer  - Unfortunately it seems that Microsoft in their ever-infinite wisdom have done away with some of the more useful logs....! 
Still digging on my end (admittedly not as much as I'd like) to see if I can find anything but haven't seen anything of yet.

**Comment by tobiasssandberg** [comment]
I saw that @ChadSimmons was able to do it with the DeliveryOptimizationDashboard.json in Pull request #18. So it can hopefully be done with this aswell.
